src/screens/main/tokens/send/CoinSend.js

```diff
@@ -105,7 +102,7 @@ class CoinSend extends Component {
     if(response.flag) {
       this.setState({maliciousCheck: true});
     } else {
-      const amountString = `${  this.state.value  }`;
+      const amountString = `${this.state.value}`;
```

Comment:
const amountString = this.state.value.toString()

Reply:
@GregTheGreek fixed